module/input/SensorFilter/data/config/SensorFilter.yaml

```diff
@@ -84,3 +84,5 @@ initial_rpy: [0, 0.3839, 0]
 
 # Specify which filtering method to use, either UKF, KF or MAHONY
 filtering_method: MAHONY
+
+urdf_path: "/home/nubots/nugus.urdf"
```

Comment:
I don't think this will work if SensorFilter is in the role but not Kinematics
For now maybe just make sure all the roles with SensorFilter have Kinematics, but I can put in a PR later to do something similar to the scripts where the urdf is always copied over.